### PR TITLE
Druid single server - minor adjustments

### PR DESCRIPTION
**Pull request by Sybrand**
Minor improvements for easier testing, debugging, and volume management.

### CODE REVIEW

File: druid_setup/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ENV_FILE=.env
 ENV_COMMON=environment/common.env
 
-PROJECT_NAME=harmony
+PROJECT_NAME=harmony-druid
```
Comment (Sybrand) [inline druid_setup/Makefile:4]: Need to clearly differentiate from the harmony web deployment.

File: druid_setup/Makefile [hunk 2]
```diff
@@ -17,6 +17,12 @@ single_server_down:
 		docker compose --project-name $(PROJECT_NAME) \
 		--env-file environment/common.env down
 
+single_server_logs:
```
Comment (Sybrand) [inline druid_setup/Makefile:20]: Convenience script for getting at logs.

File: druid_setup/single/docker-compose.yml [hunk 3]
```diff
@@ -33,14 +34,13 @@ volumes:
     driver_opts:
       type: none
       o: bind
-      device: /home/share
+      device: ${DRUID_SHARED_FOLDER:-/home/share}
```
Comment (Sybrand) [inline druid_setup/single/docker-compose.yml:37]: Makes it a bit easier to test locally

File: druid_setup/single/docker-compose.yml [hunk 4]
```diff
@@ -52,9 +52,12 @@ services:
       - ZEN_DRUID_VERSION=${DRUID_VERSION}
 
   postgres:
-    container_name: postgres
     image: postgres:latest
     restart: always
+    ports:
```
Comment (Sybrand) [inline druid_setup/single/docker-compose.yml:57]: Make it a bit easier to get at the postgres server by exposing the port.

**Review by Ideonvn — APPROVED** [review]
Looks good. The project name default I can argue against a bit, but not worth the hassle at this point.

File: druid_setup/single/docker-compose.yml [hunk 5]
```diff
@@ -22,6 +22,7 @@ volumes:
   zen_extensions: {}
   zookeeper_data: {}
   zookeeper_datalog: {}
+  zookeeper_logs: {}
```
Comment (AbbyBeeler) [inline druid_setup/single/docker-compose.yml:25]: for consistency's sake same changes in clustered? https://github.com/Zenysis/Harmony/tree/master/druid_setup/cluster

**Review by AbbyBeeler — APPROVED** [review]

File: druid_setup/single/docker-compose.yml (hunk 5 above)
Comment (Sybrand) [reply druid_setup/single/docker-compose.yml:25]: Hmmm.... good point, but I'm going to skip that. The scope of this PR is really restricted to single server, and I don't have an easy way to test any changes I make to the clustered setup right now.

**Pull request merged**